src/uct/ib/rc/verbs/rc_verbs_impl.h

```diff
@@ -195,7 +195,7 @@ uct_rc_verbs_iface_fill_inl_am_sge_iov(uct_rc_verbs_iface_t *iface, uint8_t id,
         struct ibv_sge *sge; \
         (_wr)->next    = NULL; \
         sge            = (_wr)->sg_list; \
-        sge->addr      = (uintptr_t)(desc + 1); \
```

Comment:
i guess we could also wrap it in parenthesis

Reply:
I added the paranthesis! Thanks!